doc/classes/ProjectSettings.xml

```diff
@@ -1960,6 +1960,12 @@
 			If [code]true[/code], the texture importer will import VRAM-compressed textures using the S3 Texture Compression algorithm. This algorithm is only supported on desktop platforms and consoles.
 			[b]Note:[/b] Changing this setting does [i]not[/i] impact textures that were already imported before. To make this setting apply to textures that were already imported, exit the editor, remove the [code].godot/imported/[/code] folder located inside the project folder then restart the editor (see [member application/config/use_hidden_project_data_directory]).
 		</member>
+		<member name="rendering/vrs/mode" type="int" setter="" getter="" default="0">
+			Set the default VRS mode for the main viewport.
```

Comment:
```suggestion
			Set the default VRS mode for the main viewport. See [method RenderingServer.viewport_set_vrs_mode] to change this at runtime, and [enum RenderingServer.ViewportVRSMode] for possible values.
```

Reply:
Actually it's probably better to refer to the Viewport method/enum.

Reply:
Changed it to the `Viewport` ones :)